src/main/java/core/basesyntax/strategy/handler/ReturnHandler.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax.strategy.handler;
+
+public class ReturnHandler extends IncrementHandler {
```

Comment:
The class `ReturnHandler` extends `IncrementHandler` but doesn't implement any additional logic. According to the SOLID principles, each class should have a clear purpose. If `ReturnHandler` is just an alias for `IncrementHandler`, consider removing it and using `IncrementHandler` directly. If it should have additional logic specific to the 'return' operation, then that logic needs to be implemented inside this class.